packages/cache-utils/tests/digests.mjs

```diff
@@ -1,21 +1,19 @@
-'use strict'
```

Comment:
Test main files must be `.mjs` due to `ava` not working well in monorepos mixing CommonJS and ESM packages. Once all packages of this monorepos are switched to ESM, this file can be renamed to `.js` and this comment can be removed.